lab08-kaylee/README.md

```diff
@@ -0,0 +1,44 @@
+## Lab-08: Single-Resource API
+
+## About This Project
+
+* Build an HTTP server using the 'http' Node module.
+* Create an Object constructor used to produce instances of a resource.
+* Create a Router constructor to manage GET, POST, PUT and DELETE requests to the server.
+
+## Project Dependencies
+
+* chai, chai-http, debug and uuid
+* To install:
+  * npm chai
+  * npm chai-http
+  * npm debug
+  * npm uuid
+
+## Developer Dependencies
+
+* mocha
+* To install:
+  * npm mocha
+
+## Making Requests
+
+* Example GET request
+  * In terminal (assuming httPie installed): http get :3000/api/note name="todo" date=74
+  * Expected output:
+
+* Example POST request
+  * In terminal (assuming httPie installed): http post :3000/api/note name="todo" date=75
+  * Expected output:
+
+* Example PUT request
+  * In terminal (assuming httPie installed): http post :3000/api/note name="todo" date=75
+  * Expected output:
+
+* Example DELETE request
+  * In terminal (assuming httPie installed): http post :3000/api/note name="todo" date=75
+  * Expected output:
+
+## Biggest Roadblocks
+
+* Testing!!! I am still having a really hard time wrapping my head around testing, particularly when to use .send(), when to supply .send() with an argument (i.e. an empty object {}, or an object with info {name: 'Kaylee', date='April 28, 2017'}, or nothing at all), and when/when not to use .before().
```

Comment:
it might make more sense when we have persisting data. when we create a resource, it's easier to visualize what needs to be sent, and how .send() is the method that acts as an engine to propel the request and it's contents. with the .before, are you talking about setting up a before block to setup your testing environment for your individual requests/methods?